webapp/components/Map/leaflet-side-by-side/index.js

```diff
@@ -0,0 +1,190 @@
+var L = require('leaflet')
+require('./layout.css')
+require('./range.css')
+
```

Comment:
https://www.npmjs.com/package/leaflet-side-by-side

Reply:
could you use this package?

Reply:
I am using it, but I had to edit it because that package does not work with newer leaflet versions.